charts/jaeger/values.yaml

```diff
@@ -167,11 +179,17 @@ collector:
   ## Additional pod labels
   ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
   podLabels: {}
-  extraConfigmapMounts: []
+  extraSecretMounts: []
   # - name: jaeger-tls
   #   mountPath: /tls
   #   subPath: ""
-  #   configMap: jaeger-tls
+  #   secretName: jaeger-tls
+  #   readOnly: true
+  extraConfigmapMounts: []
+  # - name: jaeger-config
```

Comment:
what's this commented out configmount example, when would we use this?

Reply:
Collector and agent both has an option of passing config-file

https://www.jaegertracing.io/docs/1.16/cli/